src/schema/v2/schema.ts

```diff
@@ -199,6 +201,7 @@ export default new GraphQLSchema({
       myCollectionCreateArtwork: myCollectionCreateArtworkMutation,
       myCollectionUpdateArtwork: myCollectionUpdateArtworkMutation,
       myCollectionDeleteArtwork: myCollectionDeleteArtworkMutation,
+      MyCollectionDeleteArtworkImage: MyCollectionDeleteArtworkImageMutation,
```

Comment:
The mutation field name should be camelcased.

Reply:
(Also same kind of comment as https://github.com/artsy/metaphysics/pull/2879/files?file-filters%5B%5D=.ts#r541054598 , should this be named more generically?

Reply:
Even though all of the others are not?

Reply:
I think he means the field name on the left side of the colon: 

```suggestion
      myCollectionDeleteArtworkImage: MyCollectionDeleteArtworkImageMutation,
```

It looks like everything left of a colon in this block is camelCase while things on the right side of a colon are a mix.